lms/templates/navigation/bootstrap/navbar-authenticated.html

```diff
@@ -33,18 +34,28 @@
       </li>
     % endif
 
-    % if show_program_listing:
+    %if show_dashboard_tabs:
       <li class="nav-item mt-2 nav-item-open-collapsed">
         <a class="nav-link ${'active' if reverse('dashboard') == request.path else ''}" href="${reverse('dashboard')}">
           ${_("Courses")}
         </a>
       </li>
+    % if show_program_listing:
       <li class="nav-item mt-2 nav-item-open-collapsed">
         <a class="nav-link ${'active' if reverse('program_listing_view') in request.path else ''}" href="${reverse('program_listing_view')}">
           ${_("Programs")}
         </a>
       </li>
     % endif
+      <%
+        self.real_user = getattr(user, 'real_user', user)
+      %>
+      <li class="nav-item mt-2 nav-item-open-collapsed">
+          <a class="${'active ' if data and data.get('profile_user_id') is not None else ''}tab-nav-link" href="${reverse('learner_profile', args=[self.real_user.username])}">
```

Comment:
1. This needs to be outdented to align with `</a>` below.
2. Can you add something like `is_on_profile_page = data and data.get('profile_user_id') is not None` above and then use the variable here?